pyubee/__init__.py

```diff
@@ -168,6 +168,22 @@ def authenticate(self, url, username, password):
         ),
         'authenticator': Evw3226Authenticator
     },
+    'EVW321B': {
```

Comment:
Could you move it right after 'EVW320B' section, after line 153, to make it alphabetical ordered?

Reply:
Done